### PR TITLE
feat(bounce): add bounce rate and session length estimation

### PR DESCRIPTION
**Pull request by trieloff**

### CODE REVIEW

File: src/queries/rum-bounces.sql [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+bounce_rate AS (
+  SELECT COUNT(*) AS all_enter_sessions
+  FROM enter_sessions
+  UNION ALL
+  SELECT COUNT(*) AS all_enter_click_sessions
+  FROM enter_click_sessions
+),
+
+aggegate AS (
```
Comment (dbrrt) [inline src/queries/rum-bounces.sql:90]:
```suggestion
aggregate AS (
```

File: src/queries/rum-bounces.sql [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  SELECT
+    SUM(pageviews) AS pageviews,
+    SUM(visits) AS visits,
+    AVG(pagespervisit) AS pagespervisit
```
Comment (dbrrt) [inline src/queries/rum-bounces.sql:94]: I'd recommend snakecasing here for the alias to improve readability

File: src/queries/rum-bounces.sql [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    all_enter_click_sessions
+  FROM bounce_rate
+)
+
+SELECT
+  MAX(pageviews) AS pageviews,
+  MAX(visits) AS visits,
+  MAX(pagespervisit) AS pagespervisit,
+  MAX(all_enter_click_sessions) / MAX(all_enter_sessions) AS bounce_rate
+FROM aggegate
```
Comment (dbrrt) [inline src/queries/rum-bounces.sql:108]:
```suggestion
FROM aggregate```

**Review by MarquiseRosier — APPROVED** [review]

**Pull request merged**